test/language/statements/class/super/in-constructor-superproperty-evaluation.js

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2018 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-makesuperpropertyreference
+description: >
+  SuperProperty evaluation order: super() thisBinding initialization occurs first.
+---*/
+class Derived extends Object {
+  constructor() {
+    super[super()];
```

Comment:
Do we want to test the return value?

Reply:
If you think it's necessary, then sure. I reduced it to this because I was able to observe passing/failing in the runtimes you listed. We can also do both. Let me know!

Reply:
This is fine with me.